src/client/client.cpp

```diff
@@ -37,6 +37,7 @@ void Client::init(std::vector<std::string>& /*args*/)
     registerLuaFunctions();
 
     g_map.init();
+    //g_map.init();
```

Comment:
???? wtf

Reply:
This is just to force the SonarCloud check to run